pkg/datapath/loader/netlink.go

```diff
@@ -141,3 +142,15 @@ func (l *Loader) DeleteDatapath(ctx context.Context, ifName, direction string) e
 
 	return nil
 }
+
+// HasDatapath return true if the interface as a tc filter attached on the given direction.
+func (l *Loader) HasDatapath(ctx context.Context, ifName, direction string) (bool, error) {
+	args := []string{"filter", "show", "dev", ifName, direction}
```

Comment:
Out of curiosity, is there any tc filter native implementation in Go?

Reply:
Looks like Florian Lehner has one maybe: https://github.com/florianl/go-tc. IIRC, we discussed this during a recent SIG-Datapath meeting and concluded it would likely be easier to just write what we need using the netlink library.

Reply:
Ah, I just remembered that we already have a similar code which is using the native implementation: https://github.com/cilium/cilium/blob/master/cilium/cmd/cleanup.go#L462.

Reply:
Ah, great! Given this is already `ready-to-merge` and addresses a `priority/release-blocker`, I'll switch to use the native implementation in a follow up. Looks like there are several other places where we could remove calls to the `tc` binary as well.